projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -44,8 +44,8 @@ import { GoTablePage } from './go-table-page.model';
 export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
 
   @Input() loadingData: boolean = false;
+  @Input() renderBoxShadows: boolean = true;
   @Input() showTableActions: boolean = false;
-  @Input() stackHeader: boolean = false;
```

Comment:
per discussion with @grahamhency - this binding was removed because it is believed no longer necessary. It was undocumented and not used in any examples.